frontend/online-shop/src/components/header/header.component.tsx

```diff
@@ -0,0 +1,28 @@
+import { Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+import { AiOutlineShoppingCart } from "react-icons/ai";
+import { CartLink, HeaderComponent, OptionLink } from "./header.style";
+const Header = () => {
+  return (
+    <HeaderComponent>
+      <Link to="/">
```

Comment:
For this `to` configurations you should also use the previous `routes` object that I told you about